packages/pretty-format/src/__tests__/pretty-format-test.js

```diff
@@ -663,6 +679,14 @@ describe('prettyFormat()', () => {
       );
     });
 
+    it('supports Unknown element', () => {
+      expect(
+        prettyFormat(React.createElement(undefined), {
+          plugins: [ReactElement],
+        })
+      ).toEqual('<Unknown />');
```

Comment:
I have doubts about supporting this `Unknown` type since it prints this warning validation in the console

console.error node_modules/fbjs/lib/warning.js:36
      Warning: React.createElement: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in.

Opt1: remove this else [here](https://github.com/facebook/jest/blob/master/packages/pretty-format/src/plugins/ReactElement.js#L82)
Opt2: leave it as is

Let me know your thoughts